network-config-analyzer/DirScanner.py

```diff
@@ -4,17 +4,21 @@
 #
 
 import os
+import io
 from GenericTreeScanner import GenericTreeScanner
+from HelmScanner import HelmScanner
+from sys import stderr
 
 
-class DirScanner(GenericTreeScanner):
+class DirScanner(GenericTreeScanner, HelmScanner):
     """
        A class for reading yaml files from a file system path
     """
 
     def __init__(self, fs_path, rt_load=False):
         GenericTreeScanner.__init__(self, rt_load)
         self.fs_path = fs_path
+        HelmScanner.__init__(self)
```

Comment:
```suggestion
        HelmScanner.__init__(self)
        self.fs_path = fs_path
```
Initialize base class before class members